InvenTree/plugin/builtin/barcodes/mouser_barcode.py

```diff
@@ -0,0 +1,71 @@
+"""The MouserBarcodePlugin matches Mouser barcodes to supplier parts."""
+
+import logging
+
+from django.utils.translation import gettext_lazy as _
+
+from company.models import ManufacturerPart, SupplierPart
+from plugin import InvenTreePlugin
+from plugin.mixins import BarcodeMixin
+
+logger = logging.getLogger('inventree')
+
+
+class MouserBarcodePlugin(BarcodeMixin, InvenTreePlugin):
+    """BarcodePlugin for matching Mouser barcodes."""
+
+    NAME = "MouserBarcode"
+    TITLE = _("Mouser Barcodes")
+    DESCRIPTION = _("Provides support for scanning Mouser barcodes")
+    VERSION = "0.1.0"
```

Comment:
Suggestion: If it works it should be 1.0

Reply:
Done.